.github/workflows/pr-build-and-e2e-tests.yml

```diff
@@ -23,15 +23,15 @@ jobs:
 
             - name: Download and install Chromium browser.
               working-directory: plugins/woocommerce
-              run: pnpx playwright install chromium
+              run: pnpm exec playwright install chromium
```

Comment:
pnpx is deprecated and doesn't seem to work properly for these anymore. moving to pnpm exec solved failures.